src/UploadManager.php

```diff
@@ -84,6 +84,6 @@ public function getDefaultDriver()
      */
     public function setDefaultDriver($name)
     {
-        $this->app['config']['upload-handler.handler'] = $name;
+        $this->container['config']['upload-handler.handler'] = $name;
```

Comment:
Because `$this->app->make('config')` is replaced with `config()` maybe it would be better to replace `$this->app['config']` with that as well.

Reply:
I followed the upgrade guide : https://laravel.com/docs/8.x/upgrade#manager-app-property